GVRf/Framework/jni/objects/components/texture_capturer.cpp

```diff
@@ -164,8 +167,10 @@ glm::mat4 TextureCapturer::getMvpMatrix(float half_width, float half_height) {
 }
 
 void TextureCapturer::callback(int msg, char *info) {
-    Java_org_gearvrf_NativeTextureCapturer_callbackFromNative(
-            mJNIEnv, mCapturerObject, msg, info);
+    if (nullptr != mCapturerObject) {
```

Comment:
That might be correct. But it can be fixed by passing a WeakReference to the native code, and that would not require hacking the listener methods.